pkg/manager/member/tiflash_util_test.go

```diff
@@ -745,67 +731,45 @@ func TestBuildTiFlashSidecarContainers(t *testing.T) {
 					},
 				}
 			}
-			cs := buildTiFlashSidecarContainers(tc)
-			g.Expect(cs).To(Equal(test.expect))
-		})
-	}
-}
-func TestSetTiFlashConfigDefault(t *testing.T) {
-	g := NewGomegaWithT(t)
-
-	type testcase struct {
-		name   string
-		config v1alpha1.TiFlashConfig
-		expect v1alpha1.TiFlashConfig
-	}
-
-	tests := []*testcase{
-		{
-			name:   "nil config",
-			config: v1alpha1.TiFlashConfig{},
-			expect: defaultTiFlashConfig,
-		},
-		{
-			name:   "custom config",
-			config: customTiFlashConfig,
-			expect: customTiFlashConfig,
-		},
-	}
-
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			setTiFlashConfigDefault(&test.config, "", "test", "test")
```

Comment:
Note the previous code for `custom config` case
`config` and `expect` both setted as `customTiFlashLogConfig`
and no deep copy(CommonConfing and ProxyConfig is the same object for `config` and `expect`)

so `config` will always equal `expect` after calling this.